mqtt_subscriber/mqtt_redis_job.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import traceback
+from os import environ, getpid
+from datetime import datetime
+from redis import StrictRedis
+from rq import Connection, Worker, get_current_job, Queue
+from rq.decorators import job
+from mqtt_subscriber.oli_broker import MQTTConnection
+from d3a_api_client.utils import get_area_uuid_from_area_name_and_collaboration_id, \
+    list_running_canary_networks_and_devices_with_live_data
+from d3a_api_client.rest_device import RestDeviceClient
+
+
+allowed_device_names = ["DOSE/OLI_42"]
+
+
+@job('canary_mqtt')
+def start(configuration_id, live_data_device_names):
+    logging.getLogger().setLevel(logging.DEBUG)
+
+    job = get_current_job()
+    job.save_meta()
+
+    device_args = {
+        "simulation_id": configuration_id,
+        "domain_name": environ["API_CLIENT_DOMAIN_NAME"],
+        "websockets_domain_name": environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"],
+        "autoregister": True
+    }
+
+    api_clients = {}
```

Comment:
```suggestion
    mqtt_rest_api_clients_mapping = {}
```
Bit verbose but also quick to understand its purpose.

Reply:
Indeed, will rename this